demos-europe/demosplan-core
# feat: addon frontend loading

This PR enables the usage of the PDF Importer in Frontend. Basically a loading mechanism was implemented to load addons dynamically after receiving the response with the stringified addon component which will be evaluated and automatically binded to the window object. 

### How to review/test

- checkout branch
- install addons zip file (ask a friendly collegue of backend for it ;))
- run: dp:addon:install addons/demosplan-addon-demospipes.zip
- clear cache
- build ewm
- navigate to the import center -> the pdf importer should mount now

### Related PRs:
https://github.com/demos-europe/demosplan-addon-demospipes/pull/21
https://github.com/demos-europe/demosplan-addon-demospipes/pull/20
https://github.com/demos-europe/demosplan-js-addon/pull/14
